CTF/the_simpsons.md

```diff
@@ -0,0 +1,24 @@
+Challenge link: https://ctflearn.com/challenge/160
+
+
+Challenge category: Cryptography, hard 
+
+Challenge points: 80 points
+
+Challenge description:
+
+Ya know, I was thinking... wouldn't the Simpsons use octal as a base system? They have 8 fingers... Oh, right! The problem! Ummm, something seems odd about this image... https://mega.nz/#!yfp1nYrQ!LOz_eucuKkjAaDqVvz3GWgbfdKWn8BhussKZbx6bUMg
+
+Challenge solution:
+
+1. firstly, go to the link provided and download it as jpg.
+2. open the jpg with nodepad -- in case there is sth hidden 
+3. at the button there are lines of python code 
+4. decode the encoded and key using octal mentioned in the description 
+5. get encoded as 'jrjerwhzkrexar', get key as 'How much did Maggie originally cost? (Divided by 8, to the nearest integer, and then plus four)'
+6. search the ans of the qn in key online and do calculation get 110
+7. follow the key - use chr(the result of the key) get key = n
```

Comment:
Could you elaborate on this line? For example: what is chr( ) and what does it do?